sources/main.cpp

```diff
@@ -116,35 +116,43 @@ main(int argc, char* argv[]) {
     context.execute();
 
     // Traces must be re-initiliazed to append to current DynaflowLauncher log file as simulation had modified it
-    dfl::common::Log::init(options, config.outputDir(), continuePreviousLogFile);
     LOG(info) << MESS(SimulationEnded, context.basename(), timerSimu.elapsed()) << LOG_ENDL;
     LOG(info) << " ============================================================ " << LOG_ENDL;
     LOG(info) << MESS(DFLEnded, context.basename(), timerGlobal.elapsed()) << LOG_ENDL;
+    DYN::Trace::resetPersistantCustomAppenders();
```

Comment:
Why perform a reset here since the program ends ?

Reply:
solved